src/registrar/templates/base.html

```diff
@@ -62,10 +62,11 @@
 
   {% if IS_DEMO_SITE %}
   <section aria-label="Alert" >
-      <div class="usa-alert usa-alert--warning usa-alert--no-icon">
+      <div class="usa-alert usa-alert--info">
       <div class="usa-alert__body">
+        <h4 class="usa-alert__heading">New domain requests are paused</h4>
         <p class="usa-alert__text">
-        <strong>BETA SITE:</strong> We’re building a new way to get a .gov. Take a look around, but don’t rely on this site yet. This site is for testing purposes only. Don’t enter real data into any form on this site. To learn about requesting a .gov domain, visit <a href="https://get.gov" class="usa-link">get.gov</a>
+        This is the new registrar for managing .gov domains. Note that we’re not accepting requests for new .gov domains until January 2024. Follow all .gov updates at <a href="https://get.gov/updates/" class="usa-link">https://get.gov/updates/</a>.
```

Comment:
**nitpick**: Could we remove the "https://" from the visible link text and just use "get.gov/updates"?

Reply:
I know this is how the other site banners are styled, but I'm thinking in this case, because this banner uses the heading as well as the paragraph, we should increase the max width of the text to span maybe like 800 or 900? I think it's currently around 500.